tests/python/pants_test/cache/test_s3_artifact_cache.py

```diff
@@ -0,0 +1,198 @@
+# coding=utf-8
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import os
+from contextlib import contextmanager
+
+import boto3
+import pytest
```

Comment:
I am excited about this contribution, glad you (re)joined the Pants community :)

Feel free to ping me if you need a committer/reviewer.